client/src/components/ItemCard/ItemCard.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import ItemCard from "./ItemCard";
+import { CartItem, emptyCartItem } from "src/interfaces";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("ItemCard Component Tests", () => {
+  let num: number = 1;
+  const testCartItem: CartItem = emptyCartItem();
+  testCartItem.item = {
+    barcode: "12345",
+    name: "test_item",
+    price: 1.0,
+    "merchant-id": "test_merchant",
+  };
+  const props = {
+    cartItem: testCartItem,
+    updateItemQuantity: (barcode: string, quantity: number) => {
+      num = quantity;
+    },
+  };
+
+  // Fresh reset dummy values for testCartItem
+  beforeEach(() => {
+    testCartItem.quantity = 1;
```

Comment:
Instead of 1, make it a constant named INIT_QUANTITY

Reply:
changed.